### ISSUE TITLE
refresh frame of buttons doesnt work

### ISSUE THREAD

**Issue by LuisBonsembiante** [photonstorm/phaser-ce#499]
* A bug in the API:
  * Phaser version(s):Phaser CE
  * What should happen: Button input.enabled= false/true take the correct frame;
  * What happens instead: Just mantain the down frame dont refresh when enabled again

**Comment by samme** [comment]
Do you know if it has ever worked that way?

**Comment by LuisBonsembiante** [comment]
I dont know but must be...if you need change the frameName in dynamic way is a bad design.

**Comment by photonstorm** [comment]
No, it has never worked like this. Button frames should be linked to input events really, so if the button is re-enabled AND the pointer isn’t over it, only then should it change the frame.

**Comment by LuisBonsembiante** [comment]
ok thanks I will change the framename manually if states change

**Comment by photonstorm** [comment]
I think it's a good idea, I just think it needs to check both things (enabled + over) before changing frame.

**Comment by LuisBonsembiante** [comment]
Thanks @photonstorm your awesome

**Comment by LuisBonsembiante** [comment]
Hi guys, Hi @photonstorm .. I have an issue for this....I implemented the solution but the button doesnt refresh the frame inmediately.. the refresh happen in some opportunities whe I move de pointer... I dont know what is the bug...could you help me?.

**Comment by LuisBonsembiante** [comment]
Hi guys any thought about this?....thanks in advance. @photonstorm

**Comment by LuisBonsembiante** [comment]
I found the error, I just replaced `element.input.enabled` for `element.inputEnabled`
